cloud-shell/cloudbuild.yaml

```diff
@@ -12,10 +12,28 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# steps:
+# - name: 'gcr.io/cloud-builders/docker'
+#   args: ['build', '-t', 'gcr.io/$PROJECT_ID/$REPO_NAME', 'cloud-shell/.']
+# - name: 'gcr.io/cloudshell-images/custom-image-validation'
+#   args: ['image_test.py', '--image', 'gcr.io/$PROJECT_ID/$REPO_NAME']
+# images: ['gcr.io/$PROJECT_ID/$REPO_NAME']
+# timeout: '1200s'
+
+# starting point for building all images re: Dan's comment
 steps:
 - name: 'gcr.io/cloud-builders/docker'
-  args: ['build', '-t', 'gcr.io/$PROJECT_ID/$REPO_NAME', '.']
-- name: 'gcr.io/cloudshell-images/custom-image-validation'
```

Comment:
Do you think this part is important? Should we try to integrate the validation tests into this pipeline?

Reply:
If you want to run the validation tests inline, you should be able to use this: `docker run -t gcr.io/cloudshell-images/custom-image-validation image_test.py --image  gcr.io/$PROJECT_ID/cloud-shell:$$TAG`

You may want to run that either after each push (would halt the build if it fails), or as a separate step (looping through each TAG again to verify each)